app/bundles/PageBundle/Translations/en_US/messages.ini

```diff
@@ -79,7 +79,7 @@ mautic.page.graph.pie.new.vs.returning.returning="Returning"
 mautic.page.graph.pie.time.on.site="Time on site"
 mautic.page.header.edit="Edit Page - %name%"
 mautic.page.header.new="New Page"
-mautic.page.help.alias="Letters and numbers (hyphens allowed) used for URL generation of this landing page. A unique alias based on the title will be autogenerated if left empty."
+mautic.page.help.alias="Letters and numbers (hyphens allowed) are used for URL generation of this landing page. A unique alias based on the title will be autogenerated if left empty."
```

Comment:
This one won't make sense in the context where it's used

Reply:
reverted